### ISSUE TITLE
core-suggest: voiceover interrupts input info with live-region content in safari 16.1 on OS X ventura (13.0)

### ISSUE THREAD

**Issue by skjalgepalg** [nrkno/core-components#664]
Issue presents when using VoiceOver in safari (16.1) with OS X Ventura (13.0)

Note: Unable to reproduce using VoiceOver  in Safari (16.0) and safari Tech preview (Release 156, Safari 16.4) with OS X Monterey (12.6)

Erroneous behavior: VoicOver interrupts declaration of element info with info about available suggestions when there is static content.

Expected behavior: VoicOver does not interrupt declaration of element with info regarding available suggestions.

**Comment by skjalgepalg** [comment]
Adopting a wait-and-see approach in case this may be solved in Ventura 13.1 release
